### PR TITLE
Simplify endpoint message deployment.

### PR DESCRIPTION
**Pull request by dzeckelev**
Resolves BV-345, BV-346.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -96,6 +99,8 @@ func main() {
 	reporter := bugsnag.NewClient(conf.Report, db, logger)
 	logger.Reporter(reporter)
 
+	vpncli.Dir = conf.VPNClient.ConfigDir
```
Comment (ababo) [inline main.go]: Please don't use global variables. Packages must be reentarable.

File: messages/ept/config/client.go [hunk 2]
```diff
@@ -43,14 +36,25 @@ const (
...
-	PingRestart:   defaultPingRestart,
-	Port:          defaultServerPort,
-	Proto:         defaultProto,
-	ServerAddress: defaultServerAddress,
+var (
+	// Dir information about the location of the directory in which
+	// the directories with configuration files are stored.
+	Dir string
+
+	defaultConfig = &cConf{
```
Comment (ababo) [inline messages/ept/config/client.go]: Please use write `NewConfig()` which fills default values.

File: messages/ept/config/client.go [hunk 3]
```diff
@@ -66,51 +70,40 @@ type cConf struct {
...
-		return "", ErrInput
-	}
-
-	switch r := record.(type) {
-	case *data.Channel:
-		if r.ID == "" {
-			return "", ErrInput
-		}
+// endpoint.Channel and endpoint.Password parameters.
+func Deploy(db *reform.DB, endpoint string) error {
```
Comment (ababo) [inline messages/ept/config/client.go]: Please rename with `DeployConfig`.

Comment (ababo) [reply messages/ept/config/client.go]: Please add an exported function which takes an endpoint ID and returns a config path. And reuse it in your code. Make sure `golint` doesn't produce warnings.

File: messages/ept/service_test.go [hunk 4]
```diff
@@ -142,6 +144,13 @@ func TestValidEndpointMessage(t *testing.T) {
 	fxt := newFixture(t)
 	defer fxt.clean()
 
+	fxt.product.ServiceEndpointAddress = &strings.Split(
```
Comment (ababo) [inline messages/ept/service_test.go]: Please add a check for the split result (not to panic on empty slice).

File: main.go (hunk 1 above)
Comment (dzeckelev) [reply main.go]: I need to change the implementation of ClientAfterEndpointMsgSOMCGet
Do you mind if I do this?

Comment (ababo) [reply main.go]: Why not to have a type with config instead of a plain function? You create it in `main.go` and pass the instance to the job worker.

Comment (dzeckelev) [reply main.go]: I meant it. What we see here is the result of our early arrangements. I delegate this functional to  the job worker.

File: messages/ept/config/client.go [hunk 5]
```diff
@@ -66,51 +56,54 @@ type cConf struct {
...
+		PingRestart:   defaultPingRestart,
+		Port:          defaultServerPort,
+		Proto:         defaultProto,
+		ServerAddress: defaultServerAddress,
+	}
+}
+
+// NewConfig creates a default VPN client configuration.
+func NewConfig() *Config {
+	return &Config{""}
```
Comment (ababo) [inline messages/ept/config/client.go]: Maybe `/etc/openvpn/config` or something similar?

File: messages/ept/service_test.go (hunk 4 above)
Comment (ababo) [reply messages/ept/service_test.go]: You still didn't add any check.

Comment (dzeckelev) [reply messages/ept/service_test.go]: I added this here
https://github.com/Privatix/dappctrl/blob/3c438a38bef5048aec327817cbd02277a8a997fb/messages/ept/service_test.go#L122

Comment (dzeckelev) [reply messages/ept/service_test.go]: I added a simple slice check

Comment (ababo) [reply messages/ept/service_test.go]: Please add some reasonable path default value in my comment above. I approve.

**Pull request merged**